app/src/main/java/com/cornellappdev/coffee_chats_android/AboutFragment.kt

```diff
@@ -0,0 +1,37 @@
+package com.cornellappdev.coffee_chats_android
+
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import kotlinx.android.synthetic.main.fragment_about.*
+
+class AboutFragment : Fragment() {
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        return inflater.inflate(R.layout.fragment_about, container, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        send_feedback.setOnClickListener {
+            visitWebsite("https://forms.gle/t9umWjckEs4NNWNS8/")
```

Comment:
Consider putting these in uppercase CONST variables. Not necessary, but very much an option :D